packages/manager/modules/pci/src/projects/project/edit/remove/remove.controller.js

```diff
@@ -42,18 +43,26 @@ export default class {
     });
   }
 
+  get isSavingsPlanAvailable() {
+    return this.pciFeatures.isFeatureAvailable(
+      PCI_FEATURES.PRODUCTS.SAVINGS_PLAN,
+    );
+  }
+
   checkSavingsPlan(serviceId) {
-    this.$http({
-      url: `/services/${serviceId}/savingsPlans/subscribed`,
-    }).then(({ data }) => {
-      this.isLoading = false;
-      if (data.length) {
-        const activePlan = data.some((p) => p.status === 'ACTIVE');
-        if (activePlan) {
-          this.canDeleteProject = false;
+    if (this.isSavingsPlanAvailable()) {
+      this.$http({
+        url: `/services/${serviceId}/savingsPlans/subscribed`,
+      }).then(({ data }) => {
+        this.isLoading = false;
```

Comment:
in case of `error`, is there an infinite loading state?

Reply:
In case of error, the `then` is not called so it causes the isLoading to be infinite to `true`